x-pack/solutions/observability/plugins/observability/public/components/loading_observability.tsx

```diff
@@ -14,7 +14,11 @@ export function LoadingObservability() {
   const { ObservabilityPageTemplate } = usePluginContext();
 
   return (
-    <ObservabilityPageTemplate pageSectionProps={{ alignment: 'center' }} showSolutionNav={false}>
+    <ObservabilityPageTemplate
+      pageSectionProps={{ alignment: 'center' }}
+      showSolutionNav={false}
+      style={{ height: '100vh', position: 'fixed', inset: 0 }}
```

Comment:
Congratulations on your first PR 🎉  

If you check `check_types.sh` job in the CI you will see that it failed due to an invalid `style` property.

In that case, I would suggest using `css` prop as advised [here](https://github.com/elastic/eui/discussions/6828#discussioncomment-7038434). Other than that, the solution looks good to me!


```suggestion
      css={css`
        height: 100vh;
        position: fixed;
        inset: 0;
      `}
```